content/coordination/onboarding/index.en.md

```diff
@@ -31,25 +31,25 @@ timely.
     -   Name(s) of organiser(s) and their email address (to be added to
         the organisers slack).
 
--   Search our database on GitHub
-    ([https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv](https://github.com/rladies/starter-kit/blob/master/Current-Chapters.csv))
-    to make sure the city does not have a chapter yet.
+-   Make a search on GitHub
+    ([https://github.com/rladies/rladies.github.io/tree/main/data/chapters](https://github.com/rladies/rladies.github.io/tree/main/data/chapters))
+    to make sure the city does not have a chapter yet. You can use CTRL+F to search using the city's name.
 
--   If there is a chapter already, check the name of the organisers
+-   If there is a chapter already, check the name of the organizers
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reply:
```suggestion
-   If there is a chapter already, check the name of the organisers
```